packages/node/src/BacktraceNodeRequestHandler.ts

```diff
@@ -24,15 +30,8 @@ export class BacktraceNodeRequestHandler implements BacktraceRequestHandler {
         'Transfer-Encoding': 'chunked',
     };
 
-    constructor(
-        private readonly _options: {
-            url: string;
-            token?: string;
-            timeout?: number;
-            ignoreSslCertificate?: boolean;
-        },
-    ) {
-        this._timeout = this._options.timeout ?? DEFAULT_TIMEOUT;
+    constructor(private readonly _options?: BacktraceNodeRequestHandlerOptions) {
+        this._timeout = this._options?.timeout ?? DEFAULT_TIMEOUT;
```

Comment:
small nit: 

maybe we should destruct options and assign timeout/ignoreSslCertificate variables instead of keeping the _options object